torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -275,7 +275,7 @@ def autograd_cache_key(
     """
     check_cacheable(gm)
     details = AOTAutogradCacheDetails(gm, example_inputs, config, fx_config)
-    pickler = AOTAutogradCachePickler()
+    pickler = AOTAutogradCachePickler(gm)
```

Comment:
Are you enabling this for AOTAutogradCache too, or is that another PR? 

If so, Dynamo produces triton_kernel_wrapper_mutation or a triton_op and I don't see those being handling here

Reply:
This PR is just FXGraphCache, I needed to update the pickling so this is just conforming the signature. I can do AOTAutograd as a follow up